.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,26 +1,70 @@
----
-name: Bug report
-about: Create an issue to fix a bug
----
-
-<!--
-BEFORE SUBMITTING: Please search to make sure this issue has not been opened already
--->
-
-## Description
-
-## Environment
- - Browser:
- - Wallet: MetaMask
- - Safe:
- - Environment:
-   - production (rinkeby)
-
-## Steps to reproduce
- 1. Go to
-
-## Expected result
-
-## Obtained result
-
-## Screenshots
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: Bug description
+    attributes:
+      label: Bug description
+      description: A short description of the bug
+      placeholder: Tell us what you see!
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: Steps
+    attributes:
+      label: Steps to reproduce
+      description: Help us reproduce the issue
+      placeholder: * Go to ...
+      value: "* Go to..."
+    validations:
+      required: true
+  - type: dropdown
+    id: browser
+    attributes:
+      label: What browser are you seeing the problem on?
+      value: Chrome
+      options:
+        - Chrome
+        - Firefox
+        - Safari
+  - type: dropdown
+    id: wallet
+    attributes:
+      label: Ethereum wallet
+      value: MetaMask
+      options:
+        - MetaMask
+        - WalletConnect
+        - Ledger
+        - Trezor
+        - Other
```

Comment:
We should probably include a 'please specifiy' field here.

Reply:
I don't think it's possible, but they can always write it in the other text fields.